docs/setup.md

```diff
@@ -49,13 +49,42 @@ sudo apt-get install \
     mlton-compiler haskell-stack
 ```
 
+### macOS packages
+
+First install Homebrew from [their website][homebrewwebsite]. Most of the dependencies
+can be installed with
+
+``` brew install git libxml2 ncurses librsvg dtc cmake ninja texlive rysnc python ccache zstd
+```
+
+The installation of mlton on Apple silicon is currently not well supported. One
+approach would be to compile mlton on another architecture and transfer it.
+
+To install the cross-compilers, run
+
+```
+brew install --cask gcc-arm-embedded
+
+brew tap messense/macos-cross-toolchains
+brew install x86_64-unknown-linux-gnu aarch64-unknown-linux-gnu
+
+brew tap riscv/riscv
+brew install riscv-tools
+```
+
+Note that CMake will require the x86 compiler before it can be invoked.
+
+The instructions in the sections below should apply for macOS.
```

Comment:
```suggestion
The instructions in the sections below should apply for both, Linux and macOS.
```

Reply:
Actually, I'd prefer it without the comma, if you don't mind

Reply:
Happy to leave the comma up to you :-)